src/helpers.js

```diff
@@ -31,11 +33,19 @@ export const getStatusType = statusCode => {
 export const getRequestDuration = startTime => new Date().getTime() - startTime
 
 /**
- * Gets the time in ms since the request has been enqueued in the web server
+ * Gets the time in ms since the request has been enqueued in the web server.
+ * It is assumed that the `queueHeader` has the value in the format `t=123456.7890` where `123456.7890` is the
+ * time in ms since epoch.
  *
  * @return {Number} The time in ms since the request has been enqueued
  */
 
-export const getQueueDuration = (req, queueHeader) => req.header(queueHeader)
-  ? new Date().getTime() - req.header(queueHeader)
-  : null
+export const getQueueDuration = (req, queueHeader) => {
```

Comment:
@jfresco log for the win! Si loggeas lo que te llega (con log level alto, tipo debug si querés) podés ver que todo tenga sentido fácil, no? :)